site/content/entry/citra-progress-report-2018-q2.md

```diff
@@ -0,0 +1,305 @@
+        title="Discord Rich Presence" >}}
+
+Discord Rich Presence works by creating a local connection with Citra and your
+Discord App. It ***does not*** connect to (or) send any data to discord servers.
+This option is user configurable, so you can choose to show or not to show rich
+presence in your discord status. Just head out to <kbd><samp>Emulation</samp> &rarr; <samp>Configure...</samp> &rarr; <samp>Web</samp> &rarr; <samp>Discord Presence</samp></kbd> and uncheck
+"Show current game in your Discord status".
+
+## [Set Default values for settings](https://github.com/citra-emu/citra/pull/3924) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+The majority of support requests we get are mostly due to users accidentally
+enabling or disabling some settings. But now with this implementation, we have
+default values for all our settings. This ensures that in case of any new
+settings changes, majority of the users (who haven't customized those settings)
+will now have the new default value automatically.
+
+## [Add support for encrypted games](https://github.com/citra-emu/citra/pull/4020) by [wwylele](https://github.com/wwylele)
+
+You can now directly use your encrypted game dumps in Citra, without having to
+decrypt them. This requires that the user provide their keys via a text file
+in `sysdata/aes_keys.txt`.
+
+Currently all known encryption methods are implemented except for the Seed
+crypto. Seed crypto is used in newer eshop games, which essentially uses an
+additional title-unique key that is stored outside of the ROM.
+
+## [Scripting support](https://github.com/citra-emu/citra/pull/4016) by [EverOddish](https://github.com/EverOddish)
+
+Scripting is an extremely useful feature which allows us to do things like
+debugging emulator internals, reverse engineering 3DS game internals,
+Tool-Assisted-Speedrunning (TAS), and cheat in games.
+(**Does not add actual Gameshark / AR Code cheat support**)
+
+Users can make their own scripts or use publicly available scripts to gain
+power over things like game screen display, inputs, and memory. Scripts can
+be used to modify the memory of games to get cheat-like behavior.
+
+Scripts can be used to read or modify the internal game state. EverOddish
+wrote a script that can read internal stats of the Pok&eacute;mon and
+automatically update image files for your current party. This feature is very
+useful for streamers who might want to display game information on their
+streams.
+
+## Movie - game input recording and playback ([#2882](https://github.com/citra-emu/citra/pull/2882), [#3922](https://github.com/citra-emu/citra/pull/3922)) by [danzel](https://github.com/danzel) and [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+This feature, not to be confused with video/audio recording, implements a way
+to record game inputs and plays them back. Every time the hardware reads an
+input device (buttons, touch screen, accelerometer, gyroscope, c-stick) we
```

Comment:
`we` => `the emulator`.
e.g.
`the emulator copies (when recording), or replaces (when playing) those values`